README.md

```diff
@@ -22,32 +28,37 @@ This is an out-of-the-box implementation of Wordpress.  It's an example of how c
     * Template: `cf create-service SERVICE PLAN SERVICE_INSTANCE`
     * Example: `cf create-service aws-rds shared-mysql mysql-service`
 
-3. Create a service instance of S3 storage. NB: A demo account will not be permitted to use the S3 service, so you can skip this for a basic demo.
+3. Create a service instance of S3 storage.
+
+cloud.gov does not have persistent local storage so you'll need to rely on S3 for storing any files uploaded to WordPress. Sandbox accounts cannot create S3 storage services, consider upgrading to a prototyping package if you need to do this.
+
   * View Services
      * `cf marketplace`
   * View Specific Service Plans
      * Template: `cf marketplace -s SERVICE`
      * Example: `cf marketplace -s s3`
   * Create Service Instance
      * Template: `cf create-service SERVICE PLAN SERVICE_INSTANCE`
-     * Example: `cf create-service s3 basic s3-service`
+     * Example: `cf create-service s3 basic-public s3-service`
 
 4. Copy the example `manifest.yml.example` to `manifest.yml`. Edit the `manifest.yml` file.
-  * Change the 'host' attribute to something unique for your site.
+  * Change the 'name' and 'host' attributes to something unique for your site.
   * Under "services:" change
     * "mysql-service" to the name of your MySQL service you created in Step 2.
     * "s3-storage" to the name of your S3 service you created in Step 3. Or delete this line if you're not using S3.
+  * The memory and disk allocations in the example `manifest.yml` file should be [sufficient for WordPress](https://codex.wordpress.org/Editing_wp-config.php#Increasing_memory_allocated_to_PHP) but may need to be adjusted depending on your specific needs.
 
 5. Copy the example `setup.json.example` to `setup.json`. Edit the `setup.json` file for your specific WordPress site information, plugins you want installed, and themes.
-  * **NOTE** The example includes a set of plugins that will be used to attach to your previously created S3 storage so you can store media uploads, like pictures, for your WordPress site. If you do not use these plugins, every time you deploy, it will destroy your uploaded files. 
+  * **NOTE** The example includes a set of plugins that will be used to attach to your previously created S3 storage so you can store media uploads, like pictures, for your WordPress site. If you do not use these plugins, every time you deploy, it will destroy your uploaded files.
   * See: [Setup JSON](#setup-json) for more information about the format of this file
 
-6. Deploy the app with a no start command
-`cf push --no-start`
+6. Deploy the app with a no start command with`cf push --no-start`
+
+This will download and install WordPress, your MySQL database, and all plugins but will not start the application on cloud.gov.
```

Comment:
Technically it doesn't "install" the database. I'd s/your MySQL database/configure it to find your MySQL database/ and s/and all plugins/and install plugins,/

Reply:
✅